realm/realm-annotations-processor/src/test/resources/io/realm/RealmDefaultModuleMediator.java

```diff
@@ -104,7 +104,7 @@ public String getTableName(Class<? extends RealmModel> clazz) {
     }
 
     @Override
-    public void insertToRealm(Realm realm, RealmModel type, Map<RealmModel, Long> cache) {
+    public void insert(Realm realm, RealmModel type, Map<RealmModel, Long> cache) {
```

Comment:
Minor. I would probably rename `type` -> `obj`, because for the single object case it is also the object we actually insert.